### PR TITLE
test: update benchmarks

### PR DESCRIPTION
**Pull request by PsypherPunk**
- add call to `scanner.run()`
- limit ports to the initial 1,023 standard

### CODE REVIEW

File: benches/benchmark_portscan.rs [hunk 1]
```diff
@@ -44,7 +46,8 @@ fn portscan_udp() {
         vec![],
         true,
     );
-    // Perform the actual scan or logic here if needed
+
+    let _ = block_on(scanner.run());
```
Comment (bee-san) [inline benches/benchmark_portscan.rs:50]: Why `let`? 🤔

File: benches/benchmark_portscan.rs [hunk 2]
```diff
@@ -23,17 +24,18 @@ fn portscan_tcp() {
...
         start: 1,
-        end: 60_000,
+        end: 1023,
     };
     let strategy = PortStrategy::pick(&Some(range), None, ScanOrder::Serial);
-    let _scanner = Scanner::new(
+    let scanner = Scanner::new(
         &addrs,
         10,
         Duration::from_millis(100),
```
Comment (bee-san) [inline benches/benchmark_portscan.rs:41]: The issue with this is that it creates the range, port strategy and parses IP addresses in the benchmark too. I will try to revive my laptop today but I moved this out of these functions to only benchmark the scanning :D

Comment (PsypherPunk) [reply benches/benchmark_portscan.rs:41]: Yep, I see what you mean.

I'll tinker with this but equally, if you manage to resurrect your laptop, I'll happily junk this PR.

File: benches/benchmark_portscan.rs (hunk 1 above)
Comment (bee-san) [reply benches/benchmark_portscan.rs:50]: Ahh I meant like in Python you could just call the function and not do `let`, I was wondering if Rust is the same 🤔

File: benches/benchmark_portscan.rs (hunk 2 above)
Comment (bee-san) [reply benches/benchmark_portscan.rs:41]: hahahah i am praying to the Mesa graphics drivers gods 🙏🏻

File: benches/benchmark_portscan.rs (hunk 1 above)
Comment (PsypherPunk) [reply benches/benchmark_portscan.rs:50]: TBH it's probably too many years of working with Python that made me use the `let`: if you don't assign things to a variable, sometimes you end up with random things going to stdout as they've nowhere else to go.

Pretty sure the Rust compiler is smarter than that though.

**Comment by PsypherPunk** [comment]
Closing in favour of #705.

**Pull request closed without merging**